pyanaconda/modules/boss/boss.py

```diff
@@ -124,7 +78,39 @@ def run(self):
         log.info("starting mainloop")
         self._loop.run()
 
-    def Quit(self):
+    def stop(self):
         """Stop all modules and then stop the boss."""
         self._module_manager.stop_modules()
-        super().stop_module()
+        super().stop()
+
+    @property
+    def all_modules_available(self):
+        """Are all modules available?
+
+         FIXME: This is a temporary method.
```

Comment:
I think we should note there why this is a temporary method, as we usually have such information for other #FIXME.

Reply:
Fixed.